apps/admin/controllers/approve_talk/update.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Admin
+  module Controllers
+    module ApproveTalk
+      class Update
```

Comment:
If you already have Talks module, why not adding Approve action there? 
Talks::Aprove, Talks::Decline (we are not talking about HTTP verbs here)

Reply:
You mean to create nested namespace? kind of `Admin::Controllers::Talks::Approve#update`? I agree, it makes sense. Good point

Reply:
No, I mean `Admin::Controllers::Talks#approve`